ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -380,6 +380,16 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-upi-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-user-provisioned-dns
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
```

```suggestion
      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
      FEATURE_SET: TechPreviewNoUpgrade
```
So we test this with minimal perms as well and catch anything missing.